src/types/Todo.ts

```diff
@@ -0,0 +1,14 @@
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
+  completed: boolean;
+}
+
+export type UpdatingTodo = Partial<Pick<Todo, 'title' | 'completed'>>;
+
+export enum FilterStatus {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
```

Comment:
better create separate file/folder for enums

Reply:
Agree with you, but i think for small app, will be ok to have 1 file with all types and enum